### ISSUE TITLE
configurable regular database backup 

### ISSUE THREAD

**Issue by upelectric** [tillsteinbach/VWsFriend#264]
I have a suggestion for a new feature. It would be nice if I could configure a regular backup of the database to a backup volume. Currently I have to backup the database manually with the UI.

**Comment by tillsteinbach** [comment]
I like the idea and it is probably pretty easy to implement. What would you think is necessary to configure. The storage location, interval and the number of backups to keep?

**Comment by tillsteinbach** [comment]
In the meantime you can also backup the database with somethign like this:

**Comment by upelectric** [comment]
Yeah, I think storage location, interval and number of backups to keep should be enough. In the meantime I will use the docker command.

**Comment by motorto** [comment]
@tillsteinbach if I run that command in a cronjob how can I then restore the database in a new install ?

**Comment by MaXtorBE** [comment]
> [@tillsteinbach](https://github.com/tillsteinbach) if I run that command in a cronjob how can I then restore the database in a new install ?

You can't.... i had a backup like that and was unable to get it back....    this is a solution probably for eggheads, not sporadic docker users

**Comment by tillsteinbach** [comment]
Restore would be something like this:
`cat your_dump.sql | docker exec -i your-db-container psql -U postgres`
